.deploy/prod-gcp.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ingresses": ["https://familie-dokument.intern.nav.no", "https://familie-dokument.nav.no"],
```

Comment:
`https://familie-dokument.intern.nav.no` is already in use by familie-brev 😬
[https://github.com/navikt/familie-brev/blob/master/.build/nais-prod.yaml](https://github.com/navikt/familie-brev/blob/master/.build/nais-prod.yaml)

We could potentially change the url in that repository

Reply:
We changed it now :) 
[PR](https://github.com/navikt/familie-brev/compare/99b082a84ae0...24de78c55867)